### ISSUE TITLE
monster2md-yamlStatblock-body.txt not providing actions/traits correctly

### ISSUE THREAD

**Issue by dcorbin** [ebullient/json5e-convert-cli#58]
I use this command: `5e-convert --index -o dm --monster monster.template 5etools-mirror-1.github.io/data`, where monster.template is a copy of monster2md-yamlStatblock-body.txt.  Looking at the generated code of thug.md (dm/compendium/bestiary/humanoid/thug.md), it has this 

```
"traits":
- !!dev.ebullient.json5e.qute.Trait {
  }
"actions":
- !!dev.ebullient.json5e.qute.Trait {
  }
- !!dev.ebullient.json5e.qute.Trait {
  }
- !!dev.ebullient.json5e.qute.Trait {
  }
```

So, while it has the correct number of actions and traits, it doesn't have the correct content for these items.

**Comment by ebullient** [comment]
Which version are you using? 
Which sources do you have enabled?

This looks like a mix of JSON and YAML. Can you show me the contents of your monster.template?

Thanks. ;)

**Comment by dcorbin** [comment]
v - 1.0.19, macOS.  Just installed the pre-built binary yesterday.  I have seen this with only default sources, and when I listed "PHB,DMG,SCAG,WDH,ALEE,FTD,MFF,MGELFT,MM,MPMM,TCE,VGM,XGE".

I've attached the file, but did have to rename to include ".txt". I got it by copying the file straight from the repo (online) as linked by the README

[monster.template.txt](https://github.com/ebullient/json5e-convert-cli/files/10050815/monster.template.txt)
